m4_sessions/automation/session_one/three_lauren_amy_dan_jason_chelle.md

```diff
@@ -1,3 +1,51 @@
 ## Group Three's Session Plan
 
 M4 Student Leaders: Lauren, Amy, Dan, Jason, Chelle
+
+
+# Session 1   3/17/17
+
+### ADDITIONAL TOPICS FOR DISCUSSION THAT WERE GLOSSED OVER:
```

Comment:
love all these topics! do you think you even need to watch the movie? are the articles/reflection enough to have a meaningful conversation about this topic?